plugins/Live/Model.php

```diff
@@ -466,7 +466,7 @@ public function makeLogVisitsQueryString($idSite, $startDate, $endDate, $segment
             $filterSortOrder = 'DESC';
         }
 
-        $segment = new Segment($segment, $idSite);
+        $segment = new Segment($segment, $idSite, Period\Factory::build('day', implode(',', array_filter([$startDate,$endDate]))));
```

Comment:
Note: That actually does not seem to create the correct period, as the bound parameter for the query when choosing the date range `2020-06-15,2020-06-21` are this:
```
1 = "2020-06-14 22:00:00"
2 = "2020-06-20 21:59:59"
3 = "2020-06-13 22:00:00"
4 = "2020-06-20 21:59:59"
```

So the inner query actually has other date values. Need to check why...